cookbooks/llama/python/llama.py

```diff
@@ -0,0 +1 @@
+/Users/jonathan/Projects/aiconfig/extensions/llama/python/llama.py
```

Comment:
I'm guessing this will be replaces with a requirements.txt that imports the llama extension once published?